articles/service-fabric/service-fabric-get-started-mac.md

```diff
@@ -28,77 +28,39 @@ ms.author: saysa
 You can build Service Fabric applications to run on Linux clusters using Mac OS X. This article covers how to set up your Mac for development.
 
 ## Prerequisites
-Service Fabric does not run natively on OS X. To run a local Service Fabric cluster, we provide a pre-configured Ubuntu virtual machine using Vagrant and VirtualBox. Before you get started, you need:
+Service Fabric does not run natively on OS X. To run a local Service Fabric cluster, we provide a pre-configured Docker image. Before you get started, you need:
 
-* [Vagrant (v1.8.4 or later)](http://www.vagrantup.com/downloads.html)
-* [VirtualBox](http://www.virtualbox.org/wiki/Downloads)
+* [Docker](https://docs.docker.com/docker-for-mac/install/)
 
->[!NOTE]
-> You need to use mutually supported versions of Vagrant and VirtualBox. Vagrant might behave erratically on an unsupported VirtualBox version.
->
-
-## Create the local VM
-To create the local VM containing a 5-node Service Fabric cluster, perform the following steps:
+## Download and start the docker image.
+To start the docker image having a pre-configured 5-node Service Fabric cluster, perform the following steps:
```

Comment:
You want to clarify that this image is just the plain ubuntu image (mention the right version in the docker file) plus a script that helps with the install.

Reply:
Correcting myself - Plain ubuntu image plus the service fabric binaries.